linera-chain/src/certificate/lite.rs

```diff
@@ -65,7 +65,7 @@ impl<'a> LiteCertificate<'a> {
     /// Verifies the certificate.
     pub fn check(&self, committee: &Committee) -> Result<&LiteValue, ChainError> {
         check_signatures(
-            self.value.value_hash,
+            self.value.executed_block_hash,
```

Comment:
We must sign the `CertificateKind` as well.

Maybe we need to modify the test that (I think) you recently added: Make sure that the _signed data_ of a `ValidatedBlock` certificate is different from the corresponding `ConfirmedBlock` certificate.

Reply:
Done in [Include CertificateKind when signing over](https://github.com/linera-io/linera-protocol/pull/3005/commits/754d86a3a30fd415f7b919597278ff5376bc3c27).

But thanks for noticing, I'm not 100% sure I would have remembered that.